src/prettytable/prettytable.py

```diff
@@ -44,6 +45,12 @@
 
 import wcwidth
 
+# Converts color codes to system default (for windows)
+from colorama import init as colorinit
+
+# Changed to colorinit in case function init() is ever necessary
+colorinit()
```

Comment:
And we'd only want to call (and import?) this when actually using colours. Again, we can deal with that later.

Reply:
Maybe we can split it into separate modules?